website/docs/kubernetes/minikube/restarting-your-minikube-cluster.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 7
+title: Restarting your Minikube cluster
+description: Restarting your local Minikube-powered Kubernetes cluster.
+keywords: [podman desktop, podman, containers, migrating, kubernetes, minikube]
+tags: [migrating-to-kubernetes, minikube]
+---
+
+# Restarting your local Minikube-powered Kubernetes cluster
+
+With Podman Desktop, you can restart your local Minikube-powered Kubernetes cluster.
+
+#### Procedure
+
+1. Open **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Resources**.
+1. Find the Minikube cluster to restart.
+1. Click <icon icon="fa-solid fa-repeat" size="lg" />.
+
+#### Verification
+
+1. Open **Containers**.
+1. Find the Minikube cluster that restarted.
+1. The cluster **Age** is consistent with the restart time.
+1. Open **Pods**.
+1. Find the pods that are running on your Minikube cluster.
+
+#### Workaround
+
+Minikube has no command to restart a cluster.
+Therefore, Podman Desktop stops the Minikube cluster, starts it again, and hopes for the best.
```

Comment:
Should we remove the "hopes for the best" part?

Reply:
The three sentences can be "Minikube has no command to restart a cluster, so you can: blahblah"